bundles/org.eclipse.passage.lbc.jetty/src/org/eclipse/passage/lbc/internal/jetty/JettyServerLauncher.java

```diff
@@ -57,7 +58,8 @@ public void unbindBackendRequestDispatcher(BackendRequestDispatcher dispatcher,
 	public LicensingResult launch(Map<String, Object> arguments) {
 		String source = JettyServerLauncher.class.getName();
 		if (server != null) {
```

Comment:
Please submit another ticket to rework this component to immutable form: all these checks for `null` looks poor